src/FSAL/FSAL_CORTXFS/export.c

```diff
@@ -105,7 +106,10 @@ fsal_status_t kvsfs_create_export(struct fsal_module *fsal_hdl,
 	/* Get the pNFS Role */
 	pnfs_role = get_pnfs_role(&KVSFS);
 
-	retval = cfs_fs_open(op_ctx->ctx_export->fullpath, &cfs_fs);
+	fsname_ptr = op_ctx->ctx_export->fullpath;
```

Comment:
Add a comment on why we are doing this. This will useful for next person who will be working on this